### ISSUE TITLE
fps测试

### ISSUE THREAD

**Issue by winky4111** [Yzichen/FlashOCC#67]
我发现长时序的fps基本保持不变，请问增加时序对fps基本没有影响吗？

**Comment by winky4111** [comment]
同时，我发现增加时序训练，似乎只需要更改配置文件中的multi_adj_frame_id_cfg参数，请问我的理解对吗？

**Comment by drilistbox** [comment]
因为实际部署时，长时序需要的bev特征在缓存里(之前已经推理过了)

**Comment by winky4111** [comment]
感谢回答，我还想问一下，对于longterm的fps测试应该使用benchmark_sequential.py对吗，因为我发现使用benchmark.py进行测试fps特别低。如果是这样的话，flashocc_stereo（2f)是不是也应该使用benchmark_sequential.py进行fps测试？

**Comment by Yzichen** [comment]
是的

**Comment by winky4111** [comment]
谢谢谢谢

**Comment by BozhouZha** [comment]
> 请问增加时序训练，只需要更改配置文件中的multi_adj_frame_id_cfg参数就可以了吗，还有如何使用时序对齐模块呢？

I figured out that the author of BEVDet4D did a tiny trick when training the sequential series. They didn't trigger the sequential alignment until some initial epochs to keep the training loss stable, please refer to:
https://github.com/HuangJunJie2017/BEVDet/issues/236
https://github.com/HuangJunJie2017/BEVDet/issues/275

Coincidentally, these questions were exactly raised by FlashOcc author LOL

**Comment by winky4111** [comment]
thank you!

**Comment by winky4111** [comment]
也就是说我需要更改multi_adj_frame_id_cfg参数，同时将with_prev改为True是吗

**Comment by BozhouZha** [comment]
> 也就是说我需要更改multi_adj_frame_id_cfg参数，同时将with_prev改为True是吗

不用, 虽然Flashocc作者和BEVdet作者的讨论里说的是Hook触发，但我发现BEVDet4D构造函数里就已经写成True了（应该是后改的，注释Args里写的还是Default False）
![image](https://github.com/user-attachments/assets/f1ff94a5-3827-4c6c-83f4-627b836a2193)
![image](https://github.com/user-attachments/assets/46f5ede1-c700-4c66-a237-c046c51db0d1)

直接跑python tools/train.py projects/configs/panoptic-flashocc/panoptic-flashocc-r50-depth4d-pano.py, 你就能看到时序已经载入了（上图二），这一步是在数据准备PrepareImageInputs sequential=True实现的

**Comment by winky4111** [comment]
好的好的，感谢
